plugins/module_utils/foreman_helper.py

```diff
@@ -366,22 +363,14 @@ def list_resource(self, resource, search=None, params=None):
 
         return self._resource_call(resource, 'index', params)['results']
 
-    def find_resource(self, resource, search, name=None, params=None, failsafe=False, thin=None):
+    def find_resource(self, resource, search, params=None, failsafe=False, thin=None):
         list_params = {}
         if params is not None:
             list_params.update(params)
         if thin is None:
             thin = self._thin_default
         list_params['thin'] = thin
         results = self.list_resource(resource, search, list_params)
-        if resource == 'snapshots':
```

Comment:
this is not needed anymore?

Reply:
Nope, it has been "fixed" on the server side.